samples/fips6-encrypted/simp_conf.yaml

```diff
@@ -1,224 +1,51 @@
-#========================================================================
-# simp config answers
-#
-# You can use these answers to quickly configure subsequent
-# simp installations by running the command:
-#
-#   simp config -A /PATH/TO/THIS/FILE
-#
-#========================================================================
 ---
-# === cli::is_ldap_server ===
-# Whether the SIMP server will also be the LDAP server.
-#
-cli::is_ldap_server: true
-
-# === cli::network::dhcp ===
-# Whether to use DHCP to set up your network ("static" or "dhcp").
+cli::version: "4.5.0"
+cli::is_simp_ldap_server: true
 cli::network::dhcp: static
-
-# === cli::network::gateway ===
-# The default gateway.
-cli::network::gateway: 192.168.121.1
-
-# === cli::network::hostname ===
-# The FQDN of the system.
-cli::network::hostname: puppet66.release.me
-
-# === cli::network::interface ===
-# The network interface to use to connect to the network.
-cli::network::interface: ens3
-
-# === cli::network::ipaddress ===
-# The IP address of the system.
-cli::network::ipaddress: 192.168.121.7
-
-# === cli::network::netmask ===
-# The netmask of the system.
+cli::network::gateway: 192.168.103.1
+cli::network::hostname: puppet.simp.test
+cli::network::interface: enp0s8
+cli::network::ipaddress: 192.168.103.7
 cli::network::netmask: 255.255.255.0
-
-# === cli::network::set_up_nic ===
-# Whether to activate this NIC now.
 cli::network::set_up_nic: true
-
-# === cli::set_grub_password ===
-# Whether to set the GRUB password on this system.
 cli::set_grub_password: true
-
-# === cli::set_production_to_simp ===
-# Whether to set default Puppet environment to 'simp'.
-#
-# Links the 'production' environment to 'simp', after backing up the
-# existing production environment.
 cli::set_production_to_simp: true
-
-# === cli::simp::scenario ===
 cli::simp::scenario: simp
-
-# === grub::password ===
-# The password to access GRUB.
-#
-# The value entered is used to set the GRUB password and to generate a hash
-# stored in grub::password.
-# P@ssw0rdP@ssw0rd
-grub::password: "$6$nPtYfEt.Xpq3AW54$ZAs2ZjLM5dguxg7KkHQRZje.Y2olmIjbz/yR/am3EK96jdGnPI2Fa6tziK/qtXpdjb1TBmWa2fkMcdyttagMb/"
-
-# === puppetdb::master::config::puppetdb_port ===
-# The PuppetDB server port number.
+grub::password: grub.pbkdf2.sha512.10000.18EA188B923287579DE28FF4CB41DEE8571213AC4BDD2906E55E147E87319E12B670EEE8AEB9689BA589F061D90EB3CC7B7FF2991ACF28309453F416517DBB74.63C6F328D86416B95DDBC46C388B1475B985B45491AB4D348B90632AE7DDCB077A8806CDCB70017CFC0761F903DAEDD8E5C08F7E9D37C2995C06802E92AD6EC7
```

Comment:
This is a GRUB2 password.  Should it be a legacy GRUB password instead?  Previous example was a legacy GRUB password.